tests/Makefile

```diff
@@ -17,7 +17,9 @@ REPO_ROOT ?= REPO_ROOT=$(ROOT)
 # Nose test runner configuration options.
 NOSECFG = --config .noserc
 
-test: pep8 test_unit test_functional
+test_agnostic: pep8 test_unit
```

Comment:
How about putting `pep8` as a dependency of `test_unit` and removing this target entirely?